graph-db/src/utils/cmr/getEchoToken.js

```diff
@@ -0,0 +1,25 @@
+const { getSecureParam } = require('../getSecureParam')
+
+/**
+ * Fetch token for CMR requests
+ * @returns {String} ECHO Token for CMR requests
+ * @throws {Error} If no token is found. CMR will not return anything
+ * if no token is supplied.
+ */
+exports.getEchoToken = async () => {
```

Comment:
Any reason youre not using the ES6 `export const ...` pattern?

Reply:
Because I'm lazy and didn't want to configure babel 🙂